doc/signing-commits.md

```diff
@@ -0,0 +1,61 @@
+# Signing commits
```

Comment:
I am not sure that commits signing is required for Hyperledger.

Reply:
Commit signing is required for Hyperledger `git commit -s`

Reply:
Ah...I see, what you mean. I have never used the `-S` option on my commits

Reply:
Yes, ```-s``` is required, but we don't need all of this GPG-related staff. So i suggest to remove it from instruction to don't scare possible contributors.

Reply:
I just removed everything related to GPG signatures. Thanks for the clarification, I had heard from a variety of different people different opinions on what was actually required.